packages/rancher-cis-benchmark/charts/templates/rbac.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: rancher-cis-benchmark
+    app.kubernetes.io/instance: release-name
+  name: cis-operator-role
+rules:
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
Is this right ? It gets * on * ?

Reply:
This is carried forward from the CIS v1 system-chart https://github.com/rancher/system-charts/blob/dev-v2.4/charts/rancher-cis-benchmark/0.1.2/templates/rbac.yaml#L11-L26